docs/streams/quickstart.html

```diff
@@ -200,7 +200,7 @@ <h4><a id="quickstart_streams_start" href="#quickstart_streams_start">Step 4: St
 <pre class="line-numbers"><code class="language-bash">&gt; bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 \
     --topic streams-wordcount-output \
     --from-beginning \
-    --formatter kafka.tools.DefaultMessageFormatter \
+    --formatter org.apache.kafka.tools.consumer.DefaultMessageFormatter \
```

Comment:
It seems to me this is a kind of broken compatibility. We should allow user to keep using `kafka.tools.NoOpMessageFormatter`, `kafka.tools.DefaultMessageFormatter`, and `kafka.tools.LoggingMessageFormatter`, but we also display warning messages to say those "strings" will be removed in 4.0

@mimaison WDYT?

Reply:
I agree with this. Many scripts could be affected by this breaking change. We should implement a deprecation cycle.

Reply:
@brandboat Could you please file another PR to fix broken compatibility? Let this PR focus on docs updates :smile

Reply:
Filed JIRA https://issues.apache.org/jira/browse/KAFKA-16795. Will file another PR soon.

Reply:
Sure we can re-add these formatter classes and have them print a deprecation message when used. 

`DefaultMessageFormatter` is the default value so we could even remove the `--formatter <>` flag from the command.

Reply:
> Filed JIRA https://issues.apache.org/jira/browse/KAFKA-16795. Will file another PR soon.

gentle ping @mimaison, I just filed https://github.com/apache/kafka/pull/16020, would be my honor if you can take a look.

> DefaultMessageFormatter is the default value so we could even remove the --formatter <> flag from the command.

Sounds good, thanks for the review 😃